platform/jobs/edxPlatformQualityMaster.groovy

```diff
@@ -33,7 +33,7 @@ archiveReports += 'edx-platform*/test_root/log/*.log, edx-platform*/test_root/lo
 archiveReports += 'edx-platform*/**/nosetests.xml,edx-platform*/**/TEST-*.xml'
 
 String htmlReports = 'pylint/*view*/, pep8/*view*/, jshint/*view*/, python_complexity/*view*/,'
-htmlReports += 'safecommit/*view*/, safelint/*view*/, eslint/*view*/'
+htmlReports += 'xsscommitlint/*view*/, xsslint/*view*/, eslint/*view*/'
```

Comment:
Actually, it doesn't break the build.  So I would vote we just go ahead and I'll communicate about the transition:

Here is a link to my PR that changes where the report goes:
https://github.com/edx/edx-platform/pull/15436
Here is the Jenkins build that passed, but just fails if you try to use that tab in the Quality Report:
https://build.testeng.edx.org/job/edx-platform-quality-pr/36506/Quality_Report/